source/vscode/src/debugger/session.ts

```diff
@@ -776,11 +782,12 @@ export class QscDebugSession extends LoggingDebugSession {
       variables: [],
     };
 
-    const handle = this.variableHandles.get(args.variablesReference);
+    const [handle, frameID] = this.variableHandles.get(args.variablesReference);
```

Comment:
See comment above, this would become a real type so that we don't have an extra int in invalid cases